musonant/questioner
# Add user object to the response for signup and login

#### What does this PR do?
- Attach user object to the signup and login responses
#### Description of Task to be completed?
- Query for the records of a user when they login or signup and
- Returning as part of the response, alongside the user token
#### How should this be manually tested?
- After cloning repository and installing dependencies,
- Access the login route: POST localhost:8000/api/v1/auth/login
- Provide valid email and password
- The response should include a user object and the token
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/163684770
#### Screenshots (if appropriate)
#### Questions:`
